.vscode/settings.json

```diff
@@ -1,3 +1,6 @@
 {
-    "typescript.tsdk": "node_modules/typescript/lib"
+    "typescript.tsdk": "node_modules/typescript/lib",
+    "yaml.schemas": {
```

Comment:
Let's revert this maybe? You should be able to have this in your own settings?

Reply:
I've left this in here because I think everyone appreciates having autocomplete and validation, not just me :p

Reply:
Sure but this is an absolute path on your computer ;)

Reply:
oh 🤦 I'll remove it